build_guest

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -fvx 
+
+wget https://cdn.kernel.org/pub/linux/kernel/v6.x/linux-6.12.5.tar.xz
+mkdir -p linux
```

Comment:
I've got mixed feelings on how to approach this -- inherently I don't like checking out into the source path which I think this will do by default.  If we were to do this, I think we need a .gitignore rule for the linux directory.  But the other thing to do is have an environment variable for LINUX_SRC which we build and checkout into.  This would better support my typical dev-container environment where we could just specify this as a /workspace thing and not mess with it.

I think doing a git checkout (or sub-module) is cleaner then tar and unpack because of cruft.  You can use depth 0 to avoid downloading lots of history.

Reply:
Yes, I agree that it would be cleaner to check out the source. I will change this.